src/encoding_implementations/gaussian_cdf.jl

```diff
@@ -83,3 +83,12 @@ function decode(encoding::GaussianCDFEncoding, i::Int)
     lower_interval_bound = (i - 1)/(c)
     return SVector(lower_interval_bound, prevfloat(lower_interval_bound + 1/c))
 end
+
+# Since the μ and σ are fixed parameters, we can also just encode/decode a state vector
+# element-wise.
+function encode(encoding::GaussianCDFEncoding, x::AbstractVector{<:Real})
+    return encode.(Ref(encoding), x)
+end
+function decode(encoding::GaussianCDFEncoding, x::AbstractVector{<:Real})
+    return decode.(Ref(encoding), x)
```

Comment:
This to me appears to violate the `encode` interface, which states that the `encode` function returns an integer. this does not return an integer.

Reply:
> This to me appears to violate the encode interface, which states that the encode function returns an integer. this does not return an integer.

You are right. For an `n` element state vector `x`, there will be `c` different possible integers for each `xi`, so this should also use the cartesian-linear mapping, so that the resulting symbol is an integer